frontend/src/components/Combobox.css

```diff
@@ -14,9 +14,9 @@
   content: "pla";
 }
 
-.usa-combo-box__list-option[data-value="#c05600"]::after {
-  background-color: #c05600;
-  color: #c05600;
+.usa-combo-box__list-option[data-value="#BC4D01"]::after {
+  background-color: #BC4D01;
+  color: #BC4D01;
```

Comment:
This should be using color from the theme itself, it is good for now but in the future this needs to be converted to SASS and use color("color_name") from the theme palette